src/ReadingProgress.js

```diff
@@ -32,7 +32,11 @@ class ReadingProgress extends Component {
 
 		var element = document.getElementById(item.href.substr(1));
 		if (!item.title) {
-			item.title = element.querySelector(this.titleSelector).textContent;
+			item.title = '';
```

Comment:
I think I would put this on an else statement:

if (titleElement) {
    item.title = titleElement.textContent;
}
else {
    item.title = '';
}

or maybe a ternary operator:

item.title = titleElement ? ttileElement.textContent : '';

I know this is really picky, but this way we have only one variable assignment instead of two consequent ones.